### ISSUE TITLE
Text alignment of menu items on windows 

### ISSUE THREAD

**Issue by jackmez** [saiminh/shynthesis-wptheme#25]
On Windows, Chrome the text in the menu buttons is not aligned to the middle of the button, any idea what might be causing this?

![unnamed](https://user-images.githubusercontent.com/18393982/133592587-ba10dcf4-43d7-4827-929a-d76d26c077d5.png)

**Comment by saiminh** [comment]
Damn this is hard for me to debug, as I have no access to a windows computer. 
What I can say is that when I use a system font instead of the custom font, a similar positioning as in your image happens. Which might point at maybe the custom font having very specific line-height or ascender/descender values. This in turn then might be also system-specific.
Could you check on your windows chrome machine if this exact same issue appears when using Helvetica or Arial, or something different (like almost the opposite offset or even look correct)?
If the issue is different, then this might be an issue with the font's system specific spacing. I have come across this when working with a custom made font before and the type designer could fix it. See this similar issue: https://stackoverflow.com/a/38508202
Could you check with the type designer if they know anything about this or could have a look? Maybe this is a common issue and they knwo what to do.

**Comment by saiminh** [comment]
Some more context to this:
https://iamvdo.me/en/blog/css-font-metrics-line-height-and-vertical-align

TLDR;
Windows and MacOs render fonts differently. I'm not a type designer so I can't go into detail. But in a nutshell, when reading a font file, MacOS looks at different values for Ascender/Descender metrics than Windows does. To make a font display in the same way across both Systems, you have to tweak these values, since MacOS and Windows also have different Philosophies on how to render type. 
If the font uses the same values for both, the exact same CSS will produce different results on Win vs Mac.

The proper way to fix this issue would be to ask the designer of the font to tweak it and ship a new version that displays the same across Mac/Win Systems.

My guess is this won't happen before launch, so there's two ways to deal with it:
**1. Find a middle-go-between value that looks acceptable on both, but prioritizes Windows (bcs > 50% users), and live with Mac descrepancy until font is fixed** _(I've implemented this now)_
2. Use Javascript to detect OS, add a class to the <body> of the site and create specific separate styling for Windows and Mac (downsides: adds extra JS code, not very reliable, separate styling for OS introduces more complexity = messy, harder to maintain and deal with changes later)
